codegen/src/xml.rs

```diff
@@ -155,7 +155,11 @@ fn codegen_xml_ser(ops: &Operations, rust_types: &RustTypes) {
                         }
                     } else if field.option_type {
                         g!("if let Some(ref val) = self.{} {{", field.name);
-                        g!("s.content(\"{xml_name}\", val)?;");
+                        if ty.s3_unwrapped_xml_output {
+                            g!("val.serialize_content(s)?;");
+                        } else {
+                            g!("s.content(\"{xml_name}\", val)?;");
+                        }
```

Comment:
The operation-level XML node is generated here, in `impl Serialize` but not `impl SerializeContent`.

https://github.com/Nugine/s3s/blob/400f0752272199cab25a84443e725a619f5fa027/codegen/src/xml.rs#L215-L228

Reply:
This is actually the place where I put it first, but it has a drawback: If the location is `None`, it the approach will not generate any XML. Actually, the way it is implemented currently seems to match the one at AWS. The example at smithy.io states:
```
<LocationConstraint xmlns="http://s3.amazonaws.com/doc/2006-03-01/">us-west-2</LocationConstraint>
```

The attribute indicates they leave out the inner tag, too.

Still want it the other way?

Reply:
Ok.